_pages/infrastructure/sandbox.md

```diff
@@ -25,7 +25,9 @@ Anybody in TTS can get an AWS sandbox account. Sandbox users have
 `power user` access, which means they have full privileges to all AWS services
 except Identity and Access Management (IAM).
 
-- [Request a sandbox account](https://github.com/18F/aws-admin/issues/new?template=new_sandbox_user.md&title=sandbox+account+for+[your+email])
+- Request a sandbox account:
+   - [18F](https://github.com/18F/aws-admin/issues/new?template=new_sandbox_user.md&title=sandbox+account+for+[your+email])
+   - [COE](https://docs.google.com/forms/d/1MrhLWxci95J3chE8CFTJ7tPeT_Y-xDbMk_8YYawr5zk/edit)
```

Comment:
The correct link is https://docs.google.com/forms/d/e/1FAIpQLSdn8Pjof5xWZSkopkEU6pumQlXGYfU4fMN9efmoDzffm2MW4A/viewform

Reply:
Ah, it redirected for me since I didn't have access. Thanks!

Reply:
Once is the editing link, the other is the viewing link. Since neither you or I have write access, when we go to the `/edit` URL, we get redirected to `/viewform`. Does that make sense?